server/db/models.py

```diff
@@ -147,6 +149,25 @@
     Column('is_active',     Boolean,    nullable=False)
 )
 
+coop_leaderboard = Table(
+    'coop_leaderboard', metadata,
+    Column('id',            Integer,    primary_key=True),
+    Column('mission',       Integer),
+    Column('gameuid',       Integer,    ForeignKey('game_stats')),
+    Column('secondary',     Integer),
+    Column('time',          TIME),
+    Column('player_count',  Integer)
+)
+
+coop_map = Table(
+    'coop_map', metadata,
+    Column('id',            Integer,    primary_key=True),
+    Column('type',          Integer,    nullable=False),
+    Column('name',          String),
+    Column('description',   String),
+    Column('filename',      String,     index=True)
+)
```

Comment:
Poor pep-8

Reply:
Flake8 has no problem with this

Reply:
Well, sure, python could be column based language.... :-)